adhocracy4/filters/filters.py

```diff
@@ -2,7 +2,22 @@
 from django.http import QueryDict
 
 
-class DefaultsFilterSet(django_filters.FilterSet):
+class PagedFilterSet(django_filters.FilterSet):
+    """Removes page parameters from the query when applying filters."""
+    page_kwarg = 'page'
+
+    def __init__(self, data, *args, **kwargs):
+        if self.page_kwarg in data:
+            try:
+                data.pop(self.page_kwarg)
+            except AttributeError:
+                # Create a mutable copy
+                data = data.copy()
+                data.pop(self.page_kwarg)
```

Comment:
Django is using a custom `QueryDict` which may be immutable. Immutable QueryDicts are throwing AttributeErrors if it is tried to modify them.